File: annotationprocessingtesting/src/main/java/com/tickaroo/tikxml/annotationprocessing/namespace/Child.java
package com.tickaroo.tikxml.annotationprocessing.namespace;

import com.tickaroo.tikxml.annotation.Attribute;
import com.tickaroo.tikxml.annotation.Xml;

/**
 * @author Hannes Dorfmann
 */
@Xml
public class Child {

  @Attribute(name = "m:id") int id;
}

File: annotationprocessortesting-kt/src/main/java/com/tickaroo/tikxml/annotationprocessing/namespace/Child.kt
package com.tickaroo.tikxml.annotationprocessing.namespace

import com.tickaroo.tikxml.annotation.Attribute
import com.tickaroo.tikxml.annotation.Xml

/**
 * @author Hannes Dorfmann
 */
@Xml
class Child {
    @Attribute(name = "m:id")
    var id: Int = 0
}
